src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -15,61 +15,61 @@
             [utils.re-frame :as rf]))
 
 (defn- settings-item-view
-  [data]
-  [rf/delay-render
-   [quo/category
-    {:list-type       :settings
-     :container-style {:padding-bottom 0}
-     :blur?           true
-     :data            data}]])
+  [^js data]
+  (quo/category
+   {:list-type       :settings
+    :container-style {:padding-bottom 0}
+    :blur?           true
+    :data            (.-item data)}))
```

Comment:
- Is `^js` is necessary here?
- Why data is changed here?

Reply:
here not, in 18th line is enough, what do you mean by data changed? you mean why it's a js object now? it's a good question, somehow i missed these changes in the description of the PR, it should be a separate change, optimisation of flat-list

Reply:
> what do you mean by data changed?

![image](https://github.com/status-im/status-mobile/assets/17097240/61411aee-230c-4f8b-af2d-5c40570ebeaa)

In first case, we are passing data directly but in second we are using .-Item

Reply:
we use flat list directly no wrappers anymore